test/integration/tcpIntegrationTests.js

```diff
@@ -228,6 +228,7 @@ describe('TCP/TLS/MLLP Integration Tests', () => {
     await Promise.all([
       TransactionModel.deleteMany({})
     ])
+    mockServer.close()
```

Comment:
We're still not waiting for this to complete or handling any errors which might occur during close.